html_table_parser/parser.py

```diff
@@ -52,7 +56,7 @@ def handle_endtag(self, tag):
             self._in_th = False
 
         if tag in ['td', 'th']:
-            final_cell = " ".join(self._current_cell).strip()
+            final_cell = "".join(self._current_cell).strip()
```

Comment:
I guess the HTML encoding example is more common. Anyhow, made it configurable and set the defaults as before. So nothing changes, but I will update the converter script to use those features. Please merge.